### PR TITLE
Release cleanups

### PR DESCRIPTION
**Pull request by jamadden**
Working on changes mentioned in #3 and general cleanups for release. Pending the outcome of the discussion about the ``ResourceWarning``s and version numbering; if we decide to make changes to fix ``ResourceWarning``s, I currently expect to merge this and start a new branch/PR to fix those.

* Non-dynamic dependency on ``zodbpickle`` for better support of wheels.
* Compatibility with zest.releaser (move changes to their own file, explicit version attribute, manifest and setup.py cleanups).
* Pep8 cleanup (lots of whitespace changes, try the ?w=0 query param).
* Fix possible performance regression on Python 2 (.items() vs .iteritems())

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+0.7.0 (unreleased)
+==================
+
+- Add support for PyPy, Python 2.7, and Python 3.
+  This requires the addition of the ``zodbpickle`` dependency, even on
+  Python 2.6.
+- Fixed the ``--days`` argument to ``multi-zodb-gc`` with recent
+  versions of ``persistent``.
+
+0.6.1 2012-10-08
```
Comment (mgedmin) [inline CHANGES.rst]: A more usual (and IMHO more readable) format is `0.6.1 (2012-10-08)`, but I suppose you only moved the changelog, not reformatted it.

File: MANIFEST.in [hunk 2]
```diff
@@ -1,5 +1,7 @@
 include *.txt
 include *.rst
+exclude buildout.cfg
+exclude tox.ini
```
Comment (mgedmin) [inline MANIFEST.in]: I like things like `buildout.cfg` and `tox.ini` in my sdists.  They provide useful information pertaining to the source (here's how you can build a dev env, here's how you can run tests).

I suppose nobody sane starts working on code by unpacking a tarball nowadays, it's always a git checkout, so feel free to disregard my comment.

**Comment by mgedmin** [comment]
:+1: Looks good to me.

File: CHANGES.rst (hunk 1 above)
Comment (jamadden) [inline CHANGES.rst]: I agree that's more usual. It's also the format that zest.releaser will use, so I think it makes since to make it all consistent.

**Pull request merged**